pkgs/by-name/li/litemdview/package.nix

```
@@ -29,7 +29,17 @@ stdenv.mkDerivation rec {
     pkg-config
   ];
 
-  meta = with lib; {
+  # Required for build with gcc-14
+  env.NIX_CFLAGS_COMPILE = "-Wno-error=implicit-int";
+
+  enableParallelBuilding = true;
+
+  nativeInstallCheckInputs = [
+    versionCheckHook
+  ];
+  doInstallCheck = true;
+
+  meta = {
     homepage = "https://notabug.org/g0tsu/litemdview";
```

```suggestion
    homepage = "https://codeberg.org/g0tsu/litemdview";
```
To reflect the `src` change